website/docs/reference/api/legacy/unleash/admin/features-v2.md

```diff
@@ -610,8 +610,15 @@ Transfer-Encoding: chunked
 ## Feature Variants
 
 ### Put variants for Feature Toggle {#update-variants}
+:::caution 
 
-<ApiRequest verb="put" url="api/admin/projects/:projectId/features/:featureName/variants" title="Create (overwrite) variants for a feature toggle (example data)" payload={[
+From 4.21 variants are tied to an environment. Check our Open API docs for the up-to-date docs: https://docs.getunleash.io/reference/api/unleash/features
+
+This endpoint affects all environments at once. If you only want to update a single environment, use [#update-variants-per-environment](#update-variants-per-environment) instead.
```

Comment:
Where is that link supposed to go? I can't seem to find that ID anywhere in this document? And why is the link text just the anchor?

```suggestion
This endpoint affects all environments at once. If you only want to update a single environment, use [#update-variants-per-environment](#update-variants-per-environment) instead.
```

Reply:
It was pointing to the section I added in the previous version. Now it should be pointing to the OpenAPI spec maybe